protecode/util.py

```diff
@@ -137,17 +138,17 @@ def _filter_resources_to_scan(component: cm.Component, resource: cm.Resource):
                 return True
 
         for _, components in component_groups.items():
```

Comment:
we should use `.values()` instead

Reply:
please also do this in a separate refactoring commit